src/test/java/se/sundsvall/comfactfacade/api/model/DocumentTest.java

```diff
@@ -18,10 +35,16 @@ void constructorAndGetter() {
 		final var result = new Document(name, fileName, mimeType, content);
 
 		// Assert
-		assertThat(result.name()).isEqualTo(name);
-		assertThat(result.fileName()).isEqualTo(fileName);
-		assertThat(result.mimeType()).isEqualTo(mimeType);
-		assertThat(result.content()).isEqualTo(content);
+		assertThat(result.getName()).isEqualTo(name);
+		assertThat(result.getFileName()).isEqualTo(fileName);
+		assertThat(result.getMimeType()).isEqualTo(mimeType);
+		assertThat(result.getContent()).isEqualTo(content);
+	}
+
```

Comment:
Test with builder methods?

Comment goes for all api model classes with builder methods where the builder methods are not tested.